lib/internal/util/comparisons.js

```diff
@@ -245,6 +291,7 @@ function innerDeepEqual(val1, val2, strict, memos) {
 
     const message1Enumerable = ObjectPrototypePropertyIsEnumerable(val1, 'message');
     const name1Enumerable = ObjectPrototypePropertyIsEnumerable(val1, 'name');
+    // TODO(BridgeAR): Adjust cause and errors properties for partial mode.
```

Comment:
Should this be resolved before we call the implementation RC?

Reply:
I would have removed the experimental warning with it applied :)
I can't think of more cases needing further handling.

Reply:
PTAL

I pushed another commit that addresses this, improves the error comparison performance, and I marked the algorithm as stable.